src/scripts/main.js

```diff
@@ -1 +1,68 @@
 'use strict';
+
+const firstRejectionMessage = `First promise was rejected`;
+const firstResolvedMessage = `First promise was resolved`;
+const secondResolvedMessage = `Second promise was resolved`;
+const thirdResolvedMessage = `Third promise was resolved`;
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', (i) => {
+    if (i.button === 0) {
+      resolve(firstResolvedMessage);
+    }
+  });
+
+  setTimeout(() => {
+    reject(firstRejectionMessage);
+  }, 3000);
+});
+
+const secondPromise = new Promise((resolve, reject) => {
+  document.addEventListener('mousedown', (i) => {
+    if (i.button === 0 || i.button === 2) {
+      resolve(secondResolvedMessage);
+    }
+  });
```

Comment:
The event listener for the 'mousedown' event in the second promise will resolve the promise on either a left or right mouse click. Ensure this behavior aligns with the intended functionality.